extensions/2.0/Khronos/KHR_materials_volume/README.md

```diff
@@ -0,0 +1,253 @@
+# KHR_materials_volume
+
+## Contributors
+
+* TODO: Name, affiliation, and contact info for each contributor
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec. Needs to be combined with `KHR_materials_transmission` or `KHR_materials_translucency`.
+
+## Exclusions
+
+* This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Extending Materials](#extending-materials)
+- [Properties](#properties)
+- [Thickness Texture](#thickness-texture)
+- [Refraction](#refraction)
+- [Absorption and Subsurface Scattering](#absorption-and-subsurface-scattering)
+- [Base Color and Absorption](#base-color-and-absorption)
+- [Transmission and Translucency](#transmission-and-translucency)
+- [Implementation](#implementation)
+- [Reference](#reference)
+- [Appendix: Full Khronos Copyright Statement](#appendix-full-khronos-copyright-statement)
+
+## Overview
+
+By default, a glTF 2.0 material describes the scattering properties of a surface enclosing an infinitely thin volume. The surface defined by the mesh represents a thin wall. The volume extension makes it possible to turn the surface into an interface between volumes. The mesh to which the material is attached defines the boundaries of an homogeneous medium and therefore must be manifold. Volumes provide effects like refraction, absorption and scattering. Scattering is not subject of this extension.
+
+<figure style="text-align:center">
+<img src="./figures/thin-thick-rendering.png"><br/>
+<img src="./figures/thin-thick.svg">
+<figcaption><em>Renderings of various objects (top) and corresponding top-down slice through the scene (bottom). The solid line represents the mesh. The gray area represents the volume. Thin-walled materials can be applied to open (left) and closed meshes (middle). The dashed line indicates the imaginary bounds of the infinitely thin volume. The volumetric material can only be applied to closed meshes (right), resulting in volumetric effects like refraction.</em></figcaption>
+</figure>
+
+The volume extension has to be combined with `KHR_materials_transmission` or `KHR_materials_translucency`. Light that falls onto the volume boundary may enter the volume, depending on the transmission or translucency of the surface's BSDF. Light traveling through the volume is subject to absorption by the medium's particles. When hitting the surface from inside the volume, the light may either decide to leave the volume or bounce back, depending on the material configuration and the incident direction.
```

Comment:
This section can only refer to KHR_materials_transmission.

```suggestion
In order for light to enter the volume, a non-zero `transmissionFactor` must be defined through the `KHR_materials_transmission` extension. Light traveling through the volume is subject to absorption by the medium's particles. When hitting the surface from inside the volume, the light will leave the volume, depending on the incident direction.
```

Reply:
I'd prefer to avoid language that says KHR_materials_transmission is the *only* way for light to enter a volume.  Future extensions and 3rd-party extensions can define means for light to enter volumes.